src/components/app-link/app-link.vue

```diff
@@ -1,11 +1,60 @@
-<script setup>
-  const LinkWithTrailingSlash = defineNuxtLink({
-    trailingSlash: 'append'
-  })
-</script>
-
 <template>
-  <LinkWithTrailingSlash>
-    <slot />
+  <LinkWithTrailingSlash
+    :class="{
+      body: Boolean(variant),
```

Comment:
So the font size and line height are dependent of the app link being underlined? 

Since the class is being set on the root element of the `LinkWithTrailingSlash`, which also is the root element of the `AppLink`, could we maybe add the body class on the open source projects component? Then the typography styles are more flexible.

Reply:
I would prefer only adding the body class for the underline variant. I prefer making develop intentions clear. And we're not sure if the body class is needed for another future variant. Adding it already might make other developers wonder if it breaks anything to remove the body class.